stablehlo/reference/StablehloValue.cpp

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2023 The StableHLO Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "StablehloValue.h"
```

Comment:
Let's use the more canonical `#include "stablehlo/reference/StablehloValue.h"`? (Same for other files - I think this style is used in a few other places) I was surprised to find that the unqualified path also compiles.